scripts/release/create_rc.mjs

```diff
@@ -110,7 +110,7 @@ shell(`git checkout -b ${rcBranchName} ${commit}`);
 shell(`lerna version ${packageVersion} --no-git-tag-version --force-publish --yes`);
 
 // Commit changes
-shell(`git commit -am "v${versionMMP}"`);
+shell(`git commit -am "chore: v${versionMMP}"`);
```

Comment:
Might wanna use a more descriptive commit message in general

```suggestion
shell(`git commit -am "chore: bump package versions to ${versionMMP}"`);
```

In the end this is not the commit that is referenced in the release as it will be the merged commit from the PR